sdk/storage/Azure.Storage.Files.Shares/samples/Sample01b_HelloWorldAsync.cs

```diff
@@ -35,191 +40,116 @@ public async Task UploadAsync()
             //
             // And you can provide the connection string to your application
             // using an environment variable.
-            string connectionString = ConnectionString;
 
-            // Get a reference to a share named "sample-share" and then create it
-            ShareClient share = new ShareClient(connectionString, Randomize("sample-share"));
-            await share.CreateAsync();
-            try
-            {
-                // Get a reference to a directory named "sample-dir" and then create it
-                ShareDirectoryClient directory = share.GetDirectoryClient(Randomize("sample-dir"));
-                await directory.CreateAsync();
+            // Name of the directory and file we'll create
+            string dirName = "sample-dir";
+            string fileName = "sample-file";
 
-                // Get a reference to a file named "sample-file" in directory "sample-dir"
-                ShareFileClient file = directory.GetFileClient(Randomize("sample-file"));
+            // Get a reference to a share and then create it
+            ShareClient share = new ShareClient(connectionString, shareName);
```

Comment:
Don't we need the commented out line that sets the connectionString for the README?

Reply:
This isn’t in a snippet

Reply:
My bad